frontend/app/src/App.tsx

```diff
@@ -852,6 +855,7 @@ export class App extends PureComponent<Props, State> {
       totalLoadTime: Math.round(
         (performance.now() - this.state.latestRunTime) * 1000
       ),
+      ...browserInfo,
```

Comment:
@sfc-gh-nbellante I think just flatting it this way will overwrite the existing os propery

Reply:
I think we need to prefix all properties with `browser`